tools/ci/taskcluster-run.py

```diff
@@ -10,8 +10,7 @@
 
 browser_specific_args = {
     "firefox": ["--install-browser"],
-    "servo": ["--install-browser"]
-
+    "servo": ["--install-browser", "--processes=12"]
```

Comment:
Note we know this causes test flakiness, so we might well not want this for the same reason as why we don't do it for other browsers.

Reply:
Also just a bit weary about running one browser in a different configuration than the others.

Reply:
Without this I think it's going to be too slow to run. I intentionally copied the servo CI configuration, which I assume they are happy with.

I agree we could probably do better here, but I think landing this as-is gives us the chance to figure out what better looks like.

Reply:
And that's because the product is Servo and not ServoDriver, so it's restarting the browser all the time.